docs/internals/variable_cleanup.rst

```diff
@@ -0,0 +1,47 @@
+.. index: variable cleanup
+
+*********************************
+Internals - Cleaning Up Variables
```

Comment:
```suggestion
Cleaning Up Variables
```
To be consistent with the others (check the image you pasted)

Reply:
fixed.